src/client/containers/BoardPage/BoardPage.container.js

```diff
@@ -5,67 +5,42 @@ import MemberBoardPage from './MemberBoardPage';
 import { useBoard } from './BoardProvider';
 import { useUser } from '../../firebase/UserContext';
 import { useParams } from 'react-router-dom';
+import { fetchFromDb } from '../fetchMethod/fetch';
 
 export default function BoardPage() {
-  const [errorCode, setErrorCode] = useState(null);
   const { boardInfo, setBoardInfo } = useBoard();
   const { boardId } = useParams();
   const { user } = useUser();
   const userId = user[0].id;
   const [isLoading, setIsLoading] = useState(true);
   useEffect(() => {
     const fetchingBoardApi = async () => {
-      const API_URL = `/api/users/${userId}/boards/${boardId}`;
-      try {
-        const apiResponse = await fetch(API_URL);
-        if (apiResponse.ok) {
-          const apiData = await apiResponse.json();
-          apiData[0].hasPassedDeadline = function () {
-            const deadlineDate = new Date(this.deadline);
-            const today = new Date();
-            return today > deadlineDate;
-          };
-          setBoardInfo(apiData[0]);
-          setIsLoading(false);
-        } else {
-          const errorResult = await apiResponse.json();
-          if (errorResult.error.startsWith('Incorrect entry Error')) {
-            setErrorCode('no_such_board');
-          }
-        }
-      } catch (error) {
-        throw new Error(error);
-      }
+      const API_URL = `${userId}/boards/${boardId}`;
+      const apiData = await fetchFromDb(API_URL, 'get');
```

Comment:
You can avoid the `API_URL` variable entirely here.

```suggestion
      const apiData = await fetchFromDb(`${userId}/boards/${boardId}`, 'get');
```